ReadMe.md

```diff
@@ -144,13 +151,17 @@ Beta version | ベータ版 | ベータばん |
 Closed beta | クローズドベータ | |
 Front-end | フロントエンド | |
 Management | 管理 | かんり | Of people, software, etc
+Manager | 管理者 | かんりしゃ |
 
 
 ### Business
 
 English | Japanese | Kana | Notes
 --------|----------|------|------
 Sales / Business | 営業 | えいぎょう |
+Salesperson | 営業者 | えいぎょうしゃ |
+Executive, company official | 役員 | やくいん |
+Chief technical officer | 最後技術責任者 | さいごぎじゅつせきにんしゃ |
```

Comment:
good to mention CEO, CFO etc as well?
i like this entry, never knew how to say CTO XD